tools/ci/before_install.sh

```diff
@@ -20,3 +21,6 @@ make all
 
 # Install system libraries
 sudo make install
+
+cd $TRAVIS_BUILD_DIR
```

Comment:
Interestingly without this the cwd is left as `$HOME/build/qpid-proton/build` (not sure why not just `$HOME/qpid-proton/build` but whatever) and the next `bin/setup` command fails

Reply:
oh that makes sense because of all of the other commands above

```
cd $HOME/build
...
cd qpid-proton
...
cd build
```

We should probably pushd/popd in this script, but that can be a future thing.

Reply:
Wow I've never heard of those, makes sense doing that now